src/main/java/emu/grasscutter/game/player/TeamManager.java

```diff
@@ -58,7 +52,13 @@ public class TeamManager {
 	@Transient private final Set<EntityBaseGadget> gadgets;
 	@Transient private final IntSet teamResonances;
 	@Transient private final IntSet teamResonancesConfig;
-	
+
+	private int useTemporarilyTeamIndex = -1;
+	/**
+	 * Temporary Team for tower
+	 */
+	private List<TeamInfo> temporaryTeam;
```

Comment:
Please add the @Transient annotation to `temporaryTeam` and `useTemporarilyTeamIndex` so they dont get saved to the database.

Reply:
Done